eventually/authentication/models.py

```diff
@@ -0,0 +1,145 @@
+"""
+This file contains models for the postgresql database.
+"""
+
+#pylint: disable=W0223
+#The pylint comment above is for ignoring the error below
+#Method 'get_full_name' is abstract in class 'AbstractBaseUser'/'get_short_name'
+#but is not overridden (abstract-method)
+
+from datetime import datetime
```

Comment:
Do you need this package?

Reply:
Yes, When updating user: 
Line 127: user.updated_at = datetime.now()

Reply:
it does not need to be done
`updated_at = models.DateTimeField(auto_now=True)`
because the field is auto update
https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.DateField.auto_now

Reply:
Understood. Corresponding lines removed.